app/src/main/java/io/apicurio/registry/rest/RegistryExceptionMapper.java

```diff
@@ -158,7 +169,7 @@ private static Error toError(Throwable t, int code) {
         error.setErrorCode(code);
         error.setMessage(t.getLocalizedMessage());
         error.setDetail(getStackTrace(t));
-
+        error.setName(t.getClass().getSimpleName());
```

Comment:
What would you think about sending the exception classname as a response header instead of in the payload?

Reply:
Hm, I'm not sure if that's a good idea since this is the error payload so for me, it make sense to add the error name here. Do you have any strong reason I'm not aware of?

Reply:
I was going to argue that the error classname is a detail that most clients won't care about, but then I realized that the "detail" field holds the java stack trace... so that argument wouldn't be very good.  :)

So nevermind - having it in the payload makes sense and is consistent with what we're doing already.

I'll update the OpenAPI definition.